converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java

```diff
@@ -332,7 +336,7 @@ public Checker hasMeasures(LocalizedError code, String... measureIds) {
 				return false;
 			}).count();
 
-			if (numberOfMeasuresRequired != numNodesWithWantedMeasureIds) {
+			if (!(numNodesWithWantedMeasureIds >= numberOfMeasuresRequired)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the opposite operator ("<") instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1940)